src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.PWMSparkMax;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+/**
+ * this subsystem takes the balls from the field to the funnel.

```

Comment:
```suggestion
 * This subsystem takes the balls from the field to the funnel.

```

The subsystem has more purposes, and you don't need to write "this subsystem" we know that it's a subsystem

Reply:
> we know that it's a subsystem

That's not a very nice wording.

Reply:
renamed